docs/source/faq/index.rst

```diff
@@ -137,6 +137,37 @@ our full suite of
 :ref:`scripting functions <config_reference/scripting/scripting_functions:Scripting Functions>`
 to create your own clever scraping mechanisms.
 
+...force ytdl-sub to re-download a video
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``ytdl-sub`` decides what videos have already been downloaded by entries in :ref:`the
+download archive file <config_reference/plugins:output_options>`,
+``./.ytdl-sub-...-download-archive.json``, at the top of the subscription/series/show
+:ref:`output directory <config_reference/plugins:output_options>` in the appropriate
+``overrides: / ..._directory:`` library path *and* the presence of the corresponding
+downloaded files under the same path. To force ``ytdl-sub`` to re-download an entry both
+need to be removed:
+
+- Move aside the downloaded files:
+
+  Rename or move the downloaded files, including the associated files with the same
+  base/stem name, such as ``./*.nfo``, ``./*.info-json``, etc..
+
+- Ensure ``ytdl-sub`` is not running and won't run, such as by cron:
+
+  ``ytdl-sub`` loads the ``./.ytdl-sub-...-download-archive.json`` file early, keeps it
+  in memory, and writes it back out late. If it's running or starts running while you're
+  modifying that file, then your changes will be overwritten when it exits.
+
+- Remove the ``./.ytdl-sub-...-download-archive.json`` JSON array item:
+
+  Search for the stem name, the basename without any extension or suffix, common to all
+  the downloaded files in this file and delete that whole entry, from the YouTube ID
+  string to the closing curly braces. Be ware of JSON traling commas.
+
+- Run ``$ ytdl-sub sub`` again.
```

Comment:
This will work if its the most recent video, but if its an older video that needs replacing it will break on existing by default.

This can be mitigated by adding to the command `-o "--ytdl_options.break_on_existing False`, but this will rescrape _every_ metadata. More can be added in the overrides like date range:

`-o "--ytdl_options.break_on_existing False  --date_range.before 20250101 --date_range.after 20240101 ` to only look at the upload year of 2024.

I haven't ran this command and I'm on my phone lol, probably needs a quick sanity run.

Reply:
K, revised further per [Discord discussion](https://discord.com/channels/994270357957648404/1102703969266049174/1407956781316964372). Still can't sanity test the example command.